solutions/devsprint-riccardo-1/OnboardingChallenge/Protocols/ViewCode.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  ViewCode.swift
+//  OnboardingChallenge
+//
+//  Created by Pedro Gabriel on 15/12/21.
+//
+
+import UIKit
```

Comment:
I see that this file is a way to structure how to write UIView code. I really like this: I'd like to discuss it with the rest of the team. These kinds of changes are useful only if all the team conforms to the same conventions.
 
But, at the moment, we have to remember to make our view conform to it.
Could you think about how we can automate this?

A protocol is an abstraction. Although abstractions can help us organize our work, we shouldn't introduce abstractions we don't need. Do you plan to use the `ViewCode` protocol in other ways other than just have the view conform to it (for example: do you expect to have an array of `ViewCode`)?
If we don't plan to use the protocol in a similar way, it is something we should not add to the codebase.